client/app/administration/user-details-form/user-details-form.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit, Output, EventEmitter, Input, OnChanges } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { UserService} from '../../_services/user.service';
+import { User } from '../../_services/userModel';
+
+declare var jQuery: any;
+
+@Component({
+  selector: 'app-user-details-form',
+  templateUrl: './user-details-form.component.html',
+  styleUrls: ['./user-details-form.component.scss']
+})
+export class UserDetailsFormComponent implements OnInit, OnChanges {
+@Input() title:string;
+@Input() user;
+@Input() mode;
+@Output() closeFormEvent: EventEmitter<any> = new EventEmitter();
```

Comment:
decide a better name for this event closeFormEvent and send message with each emit